hue/test_hue.py

```diff
@@ -28,6 +30,11 @@ def verify_instance(self, instance_name):
         ("HA", ["m-0"]),
     )
     def test_hue(self, configuration, machine_suffixes):
+        if self.getImageOs() == 'ubuntu':
+            if self.getImageVersion() <= pkg_resources.parse_version("1.4"):
```

Comment:
maybe change this to 
self.getImageVersion() < pkg_resources.parse_version("1.5")

Reply:
See above comment.